src/Fhir.Anonymizer.Tool/Stu3-config-draft.json

```diff
@@ -0,0 +1,267 @@
+    //{"path": "Contract.term.topic", "method": "redact"},
+    {"path": "Contract.term.text", "method": "redact"},
+    //{"path": "Contract.term.offer.answer.value", "method": "redact"},
+    //{"path": "Contract.term.offer.text", "method": "redact"},
+    //{"path": "Contract.term.offer.linkId", "method": "redact"},
+    //{"path": "Contract.term.asset.context.text", "method": "redact"},
+    //{"path": "Contract.term.asset.condition", "method": "redact"},
+    //{"path": "Contract.term.asset.text", "method": "redact"},
+    //{"path": "Contract.term.asset.linkId", "method": "redact"},
+    //{"path": "Contract.term.asset.answer", "method": "redact"},
+    //{"path": "Contract.term.asset.valuedItem.entity", "method": "redact"},
+    //{"path": "Contract.term.asset.valuedItem.payment", "method": "redact"},
+    //{"path": "Contract.term.asset.valuedItem.linkId", "method": "redact"},
+    //{"path": "Contract.term.action.linkId", "method": "redact"},
+    //{"path": "Contract.term.action.contextLinkId", "method": "redact"},
+    //{"path": "Contract.term.action.requesterLinkId", "method": "redact"},
+    //{"path": "Contract.term.action.performerLinkId", "method": "redact"},
+    //{"path": "Contract.term.action.reason", "method": "redact"},
+    //{"path": "Contract.term.action.reasonLinkId", "method": "redact"},
+    {"path": "Contract.term.group", "method": "redact"},
+    {"path": "ExplanationOfBenefit.disposition", "method": "redact"},
+    {"path": "ExplanationOfBenefit.information.value", "method": "redact"}, //newadd
+    //{"path": "ExplanationOfBenefit.preAuthRef", "method": "redact"},
+    //{"path": "ExplanationOfBenefit.supportingInfo.value", "method": "redact"},
+    {"path": "ExplanationOfBenefit.insurance.preAuthRef", "method": "redact"},
+    //{"path": "ExplanationOfBenefit.accident.location", "method": "redact"},
+    //{"path": "ExplanationOfBenefit.item.location", "method": "redact"},
+    //{"path": "ExplanationOfBenefit.addItem.location", "method": "redact"},
+    {"path": "ExplanationOfBenefit.processNote.text", "method": "redact"},
+    {"path": "ExplanationOfBenefit.benefitBalance.name", "method": "redact"},
+    {"path": "ExplanationOfBenefit.benefitBalance.description", "method": "redact"},
+    {"path": "ExplanationOfBenefit.benefitBalance.financial.allowed", "method": "redact"},
+    //{"path": "InsurancePlan.name", "method": "redact"},
+    //{"path": "InsurancePlan.alias", "method": "redact"},
+    //{"path": "InsurancePlan.coverage.benefit.requirement", "method": "redact"},
+    //{"path": "InsurancePlan.plan.generalCost.comment", "method": "redact"},
+   
+    //{"path": "Patient.where(animal.exists()).name", "method": "keep"}, ///Newadd
+    
+    
+    {"path": "nodesByType('Dosage').text", "method": "cryptoHash"} ///Newadd
+    {"path": "nodesByType('oid')", "method": "redact"}, //????????????????????
+    {"path": "nodesByType('Contributor').name", "method": "redact"},  ///Newadd

```

Comment:
where is the definition for this data type? I can't find it.

Reply:
```suggestion
    {"path": "nodesByType('Contributor').name", "method": "redact"},  ///Newadd
https://www.hl7.org/fhir/STU3/metadatatypes.html#Contributor
```

Reply:
Contributor: https://www.hl7.org/fhir/STU3/metadatatypes.html#Contributor
Dosage is a kind of type for medicationdispense resource: https://www.hl7.org/fhir/STU3/medicationdispense.html, but it seems not a datatype

Reply:
I see. They are both data types. Thanks